pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -386,8 +386,6 @@ export class SETroubleshootPage extends React.Component {
                     { title: itm.description }
                 ];
                 if (itm.count > 1) {
-                    title = cockpit.format(cockpit.ngettext("$0 occurrence", "$0 occurrences", itm.count),
-                                           itm.count);
```

Comment:
It's being used in line 406 below in the `<CardTitle>`. Indeed it makes not much sense to change it to *this* string, as the default is "SELinux access control errors". But I feel this requires a little more research what this was really meant to do?

Reply:
Explained it in the commit message what went wrong - found the falty commit in the git history.